examples/bevy_wgpu_render.rs

```diff
@@ -106,14 +110,12 @@ fn setup(
         size: Vec2::new(10., 10.),
         fov: 3.,
     }));
-    for x in 0..quad_count {
-        commands.spawn(MaterialMeshBundle {
-            mesh: mesh_handle.clone(),
-            material: material_handle.clone(),
-            transform: Transform::from_xyz((x as f32 * quad_size) + 0.5, x as f32 / 5., 0.0),
-            ..Default::default()
-        });
-    }
+    commands.spawn(MaterialMeshBundle {
```

Comment:
Just a heads up, this part will be reworked completely in the upcoming resolution of #20